submissions/tosinIshola/HTML/Ecommerce Website.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Well done @gracioust 

Is there a reason why you created this file also? If it is done by mistake, kindly remove it as part of your PR

Reply:
i changed the name of the html file

Reply:
I changed it from Ecommerce Website.html to Ecommerce.html. I don't know why it is showing

Reply:
can i close this PR and create another one?